test/python3/test_mercury_kw_login.py

```diff
@@ -4,11 +4,38 @@
 
 import common_functions as cf
 
-def test_login():
+
+def test_login_success():
```

Comment:
My 2 cents on these Python tests is that they should not be testing functionality of the thing they are wrapping. The only tests worth writing here are to ensure the wrappers are valid over the C++ library. Functionality should be tested with Googletest and inside the C++ library.

Reply:
👍 to this